pkg/relic/Makefile.include

```diff
@@ -1 +1,8 @@
 INCLUDES += -I$(PKGDIRBASE)/relic/include
+
+ifneq (,$(filter cortex-m%,$(CPU_ARCH)))
+  # jerryscript package package is not using system includes right now, so
```

Comment:
Same but with `relic` instead of `jerryscript`. :p

Reply:
yeah fixed the wording